src/activity.js

```diff
@@ -14,13 +14,14 @@
  * limitations under the License.
  */
 
-import {getElementService} from './element-service';
+import {getElementServiceForDoc} from './element-service';
+
 
 /**
- * @param {!Window} win
+ * @param {!Node|!./service/ampdoc-impl.AmpDoc} nodeOrDoc
  * @return {!Promise<!Activity>}
  */
-export function activityFor(win) {
+export function activityForDoc(nodeOrDoc) {
   return /** @type {!Promise<!Activity>} */ (
-      getElementService(win, 'activity', 'amp-analytics'));
+      getElementServiceForDoc(nodeOrDoc, 'activity', 'amp-analytics'));
```

Comment:
FYI: this is the correct use of `getElementServiceForDoc`. However, `getServicePromiseForDoc` is better in the `instrumentation.js`. The reason is that this is a public declaration of the service.